src/templates/floating-close-button.js

```diff
@@ -1,7 +1,7 @@
 export default () => {
     return (
         `<div class="jw-float-close jw-reset">` +
```

Comment:
Since we're having two divs now, we should eliminate `jw-float-close-icon` and append the icon directly to `jw-float-close`.